src/test/java/subway/StationLineRequest.java

```diff
@@ -0,0 +1,88 @@
+package subway;
+
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import static io.restassured.RestAssured.given;
+
+public class StationLineRequest {
+
+    /**
+     * 지하철 노선 목록을 조회하고 jsonPath 반환
+     *
+     * @return 지하철 노선 목록을 나타내는 jsonPath 반환
+     */
+    static JsonPath findAllStationLinesRequest() {
+        return given().log().all()
+                .when()
+                .get("/lines")
+                .then().log().all()
+                .statusCode(HttpStatus.OK.value())
+                .extract().jsonPath();
+    }
+
+    /**
+     * 주어진 지하철 노선 ID에 해당하는 지하철 노선 정보 반환
+     *
+     * @param stationLineId 지하철 노선 ID
+     * @return 지하철 노선 정보를 나타내는 jsonPath 반환
+     */
+    static JsonPath findStationLineRequest(Long stationLineId) {
+        return given().log().all()
+                .when()
+                .get("/lines/" + stationLineId)
+                .then().log().all()
+                .extract().jsonPath();
+    }
+
+    /**
+     * 지하철 노선 생성 요청 후 Response 객체 반환
+     *
+     * @param request 지하철 요청 정보를 담은 객체
+     * @return REST Assured 기반으로 생성된 Response 객체
+     */
+    static ExtractableResponse<Response> createStationLineRequest(subway.dto.StationLineRequest request) {
+        return given().log().all()
+                .body(request)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/lines")
+                .then().log().all()
+                .statusCode(HttpStatus.CREATED.value())
+                .extract();
+    }
+
+    /**
+     * 지하철 노선 수정 요청 후 Response 객체 반환
+     *
+     * @param request       지하철 수정 정보를 담은 객체
+     * @param stationLineId 수정할 지하철 노선 ID
+     */
+    static void updateStationLineRequest(subway.dto.StationLineRequest request, Long stationLineId) {
```

Comment:
fixture에 한글을 사용한 것처럼 메서드명에도 한글을 적용한다면 인수 테스트의 가독성에 도움이 될 것 같은데 어떻게 생각하시나요?
```suggestion
    static void 노선_수정_요청(subway.dto.StationLineRequest request, Long stationLineId) {
```

Reply:
아주 좋은 것 같습니다.  👍
다음 미션부터는 한글을 적극적으로 사용해보도록 하겠습니다.